start_bundle_coupled.R

```diff
@@ -178,6 +168,21 @@ deletedFolders <- 0
 
 stamp <- format(Sys.time(), "_%Y-%m-%d_%H.%M.%S")
 
+message("path_remind:           ", if (dir.exists(path_remind)) green else red, path_remind, NC)
+message("path_magpie:           ", if (dir.exists(path_magpie)) green else red, path_magpie, NC)
+message("path_settings_coupled: ", if (file.exists(path_settings_coupled)) green else red, path_settings_coupled, NC)
+message("path_settings_remind:  ", if (file.exists(path_settings_remind)) green else red, path_settings_remind, NC)
+message("path_remind_oldruns:   ", if (dir.exists(path_remind_oldruns)) green else red, path_remind_oldruns, NC)
+message("path_magpie_oldruns:   ", if (dir.exists(path_magpie_oldruns)) green else red, path_magpie_oldruns, NC)
+message("prefix_runname:        ", prefix_runname)
+message("n600_iterations:       ", n600_iterations)
+message("run_compareScenarios:  ", run_compareScenarios)
+
+if (any(! file.exists(c(path_settings_coupled, path_settings_remind))) ||
+    any(! dir.exists(c(path_remind, path_magpie, path_remind_oldruns, path_magpie_oldruns)))) {
+  stop("Missing files or directories, see in red above.")
+}
+
```

Comment:
Why did you move it downwards?

Reply:
- It needed to be below the `dir.create("output")` to make sure `path_remind_oldruns` exists.
- I first wanted to do it via `errorsfound` and fail at the end of the file, but then I thought it is better to fail immediately. So yeah, one could move it back up some lines, but I think it does not matter.

Reply:
Ok, now one medal for me. It was kind of obvious that 'output' is part of ' path_remind_oldruns' .

Reply:
Sorry, I know it is not important and we can leave it as is, but I simply can't shut up: if `output` has just been created the line above there will be nothing to be found in `path_remind_oldruns` anyway, is it?

Reply:
Yes, so no run will be found which is ok, the script does not fail in this case.